api/types/plugin.go

```diff
@@ -23,13 +23,24 @@ import (
 	"github.com/gravitational/trace"
 )
 
+// PluginType represents the type of the plugin
```

Comment:
This is added as a single primitive identifier for both the frontend/backend to refer to a type of plugin.